.gitignore

```diff
@@ -1,5 +1,5 @@
 /build/massastation/*
-
+/massastation
```

Comment:
we use task to build massastation now, please remove your local ./massastation file

```suggestion
```